### ISSUE TITLE
make strings translatable

### ISSUE THREAD

**Issue by ericfg** [crownheightsaid/mutual-aid-app#49]
Going to have a hard time getting some groups I know to use this unless the core interface can be in something other than english (based on each user's choice)

**Comment by mjmaurer** [comment]
agreed. i18n was originally included but structure changed very quickly and so was removed.

**Comment by ericfg** [comment]
wow. For me this is a major block to using this tool. 
any thoughts about getting back to your initial idea?

**Comment by mjmaurer** [comment]
https://www.i18next.com/ is what I originally setup.

It works well, but very configurable. I can try and find time this week to reintroduce it with the opinionated setup I had.

**Comment by ericfg** [comment]
thanks. I have zero js experience or I'd try to help

**Comment by mjmaurer** [comment]
#59 centralizes strings with i18next, which makes adding support for i18n easier. Next steps:
1. Dropdown in webapp for selecting language
2. Request locale for user in slackapp

**Comment by piratefsh** [comment]
@mjmaurer can i close this issue? just wanna clean up our list of issues a bit.

**Comment by mjmaurer** [comment]
While it's somewhat easy to finish implementing, this issue is still open. Need to have a language picker where we consume strings (webapp, slack, etc).
